### PR TITLE
Reformat documentation

### PR DESCRIPTION
**Pull request by AlekSi**
(and test Cloudflare Pages deployment)

Deployment URL is visible in logs:

CleanShot 2023-01-13 at 20 44 53@2x

We might make it more accessible later.

## Readiness checklist

* [ ] I added unit tests for new functionality or bug fixes.
* [ ] I added integration tests for new functionality or bug fixes.
* [ ] I added compatibility tests for new functionality or bug fixes.
* [x] I made spot refactorings.
* [x] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2022-10-06-how-to-start-ferretdb-locally-with-docker.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ date: 2022-10-06
 
 Even though FerretDB is still in development, it's possible to check out its capabilities and run it in your local environment!
 
-The process of installing and running FerretDB is not complicated at all, and you can do it in just a few steps.  
+The process of installing and running FerretDB is not complicated at all, and you can do it in just a few steps.
```
Comment (AlekSi) [inline website/blog/2022-10-06-how-to-start-ferretdb-locally-with-docker.md:16]: @Fashander Please configure your editor to trim trailing whitespace

**Review by Fashander — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Comment by AlekSi** [comment]
@Fashander @rumyantseva PTAL, enabled LFS check

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -35,7 +35,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
-          fetch-depth: 0
+          fetch-depth: 0 # for `git describe` to work
+          lfs: false # LFS is used only by website
```
Comment (rumyantseva) [inline .github/workflows/go.yml:39]: 👍

**Review by rumyantseva — APPROVED** [review]
LGTM

@AlekSi do we need blog-preview similar to docs-preview?

**Comment by AlekSi** [comment]
> do we need blog-preview similar to docs-preview?

I think that will be useful, yeah

**Pull request merged**